packages/apps/kubernetes/templates/helmreleases/monitoring-agents.yaml

```diff
@@ -45,7 +45,7 @@ spec:
     vmagent:
       externalLabels:
         cluster: {{ .Release.Name }}
-        tenant: {{ $targetTenant }}
+        tenant: {{ .Release.Namespace }}
```

Comment:
Shouldn't it be `.Release.Name`?

Reply:
no, release name - is name of HelmRelease (cluster name), like kubernetes-patch12toroot

Reply:
tenant - its name of namespace, where HelmRelease

Reply:
got it!